app/src/utils/convertDateStringToTimeAgo.ts

```diff
@@ -0,0 +1,53 @@
+const convertDateStringToTimeAgo = ({
+  date,
+  precision = 1,
+}: {
+  date: string;
+  precision?: number;
+}) => {
+  const now = new Date();
+  const then = new Date(date);
```

Comment:
not terribly hung up on it, but I'm wondering if there is a better variable name instead of `then` (especially since it's a JS keyword). Maybe `pastDate` or something?

Reply:
Yes I'll change these to nowDate and thenDate, that reads better. Though I believe `then` is not a keyword. The only use I know is for chaining promises, but not as a standalone keyword.